.gitignore

```diff
@@ -1 +1,3 @@
 *~
+.history/
```

Comment:
What is `.history` coming from? I think this can stay if it's something we expect lots of other devs to have. But if it's a detail of your particular dev setup, I think it makes sense for you to just put that in your `~/.gitignore`. If it stays though, it should have a comment about what it's for.